File: Somatorio.java
package Entregaveis;

public class Somatorio {

    public static void main(String[] args) {
        int numero = 5;
        System.out.println("A soma dos números de 1 até " + numero + " é: " + somatorio(numero));
    }

    public static int somatorio(int n) {
        int soma = 0;
        for (int i = 1; i <= n; i++) {
            soma += i;
        }
        return soma;
    }
}

File: Somatorio.js
function somatorio(n) {
    let soma = 0;
    for (let i = 1; i <= n; i++) {
        soma += i;
    }
    return soma;
}

const numero = 5; 
console.log(`A soma dos números de 1 até ${numero} é: ${somatorio(numero)}`);